docs/semantics.md

```diff
@@ -377,3 +377,38 @@ Not all of the usual file system features are supported. Most prominently:
 - File and directory permissions and ownership cannot be changed. See the permissions section above.
 - Modification times are not tracked for any inodes except for files.
 - No other times besides modification time are tracked. For example, ctime and atime are not tracked (but will be set to something reasonable). Requests to change them will appear to succeed, but the results are unspecified.
+
+# Incompatibilities with GCS (Google Cloud Storage)
+
+## Unsupported objects
+
+**Problem**: GCS supports creating objects having names or prefixes such as `A//B`
+(in fact object `A//B` can even co-exist with object `A/B` in the same
+GCS bucket). Such objects cannot be represented in a gcsfuse-mounted filesystem,
+because linux filesystem does not support directories named `/`. This causes
+either an `input/output error` or unexpected behavior on listing the directory
+entries of `A` (i.e. `ls A`) in a gcsfuse-mounted file-system. Because of
+this issue, even objects having name or prefix `A/C` (which are perfectly
+supported by linux filesystem) cannot be listed by gcsfuse by `ls A`.
+
+**Workaround**: To access objects having name or prefix `A/C` in a
+gcsfuse-mounted directory in the above case, you should either move (preferably to
+`A/B` if there is no other such object/prefix) or
+delete (NOT recommended as it causes irreversible data-loss),
```

Comment:
Instead of irreversible data-loss mention that it will delete the underlying object.